client/src/Screens/SignUpScreen.jsx

```diff
@@ -0,0 +1,172 @@
+import * as React from "react";
+import {
+  Grid,
+  Box,
+  Typography,
+  TextField,
+  Button,
+  Avatar,
+} from "@mui/material";
+import { Link } from "react-router-dom";
+import AuthenticationLayout from "../components/Layouts/AuthenticationLayout";
+
+import { useSnackbar } from "notistack";
+import { useForm, Controller } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+
+import { SignUpValidations } from "../validations/Registration";
+
+export default function App() {
```

Comment:
Change function name

```suggestion
export default function SignUpScreen() {
```